src/sortStudents.ts

```diff
@@ -1,16 +1,85 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
```

Comment:
Order should be enum too, So you no need type string 'asc' or 'desc' in all places where it used
```suggestion
export enum SortOrder {
  ACS = 'ASC';
  DESC = 'DESC'
}
```